README.md

```diff
@@ -1,2 +1,243 @@
+
+
+## How? (_Proposal_)
+
+Our "***metric***" for this should be a "hybrid" NPS/Productivity score.
+
+> Mark will formulate exactly how to calculate this and capture the data
+  from all members on an on-going basis. This is one of our USPs so getting regular data/feedback is essential.
+
+### TODO
+
++ [ ] **Market _Validation_**
+  + [ ] Create a basic survey (_using your choice of survey tool_)
+  with ***questions*** (_proposed_):
+    + [ ] Where do you get your most valuable work done?
+    + [ ] What time of day are you most productive?
+    + [ ] On a scale of 1 - 5
+    (_where 1 is **disagree strongly** and 5 is **agree strongly**_)
+    + I work best when in a quiet environment.
+    + I regularly wear earphones to block out noise
+    when I'm trying to concentrate on my work.
+    + I find it difficult to get back into my work if I am interrupted.
+    + I like to openly discuss my work with my co-workers and pick their brains without calling a "formal" meeting.
+  + [ ] Collect responses to the survey by using Google Adwords
+  to target people who are searching for "London Co-Working" direct them to the survey.
+  + [ ] Use Google Analytics to track response/dropout rate.
++ [ ] ***Membership Registration/Management*** System (_can be a [**spreadsheet**](https://github.com/dwyl/html-form-send-email-via-google-script-without-server) for MVP with a view to using our [CRM](https://github.com/dwyl/hq/issues/161) after MVP_)
+but ideally would link to the ***payment processing***/verification so it's clear who has paid.
+
++ [ ] Monthly/Pay-as-you-Go **Payment Collection** System (_e.g: GoCardless_)
+
++ [ ] **Desk allocation** system (_so people can **reserve** a **specific desk**
+  if they want to and we **know** when we're "**full**"_)
+
++ [ ] **Check-in** system when people arrive (_also useful for fire safety_)
+
++ [ ] Anonymous ***feedback*** system so people can complain if they feel the need to?
+see: https://github.com/dwyl/hq/issues/183
+
+> Making the QuiteSpace Co-working space a success
+is an _integral_ part of dwyl's wider mission.
+Having a productive place for our London-based team
+to work is _essential_ for in-person collaboration.
+
+
+### _Fixed_ Overheads (_Costs we have to Cover_)
+
+We calculate our pricing based on our need to cover ***Fixed Overheads***:
++ Rent: £2500 (_please input exact number_)
++ Rates: £500
++ Electricity: £200/month
++ Cleaning: £100/month
++ Internet: £100/month
++ Insurance: £50/month
++ "Free" Tea & Coffee: £50/month  (_please define a monthly budget_)
+
+Total _Fixed_ Costs: £3,600
+
+> _**Question**: should we **budget** for an **Office Manager**? (part time?)_ <br />
+_please discuss_: https://github.com/dwyl/quietspace/issues/25
+
+### Number of Desks
+
++ Upstairs: 16 Seats, 2 Standing Desks
++ Downstairs: 12 Seats, 2 Standing Desks
+
+### _Potential_ Revenue at "_Full Capacity_"
+
+32 Places @ £170/month = **£5,440**
+
```

Comment:
This figure includes the desks that will be used by dwyl. Which makes sense, but we need to be aware that 8-12 of these places will not be bringing in revenue.